components/Login.tsx

```diff
@@ -53,39 +55,131 @@ export default function Login({ setCredentials }: LoginArgs) {
+            <form className='loginForm' onSubmit={onSubmit} style={{ display: loginMethod === 'email' ? 'block' : 'none' }}>
+                <Callout kind={CalloutKind.Warning}>This website is <b>NOT</b> an offical VanMoof service/product</Callout>
+                <Callout kind={CalloutKind.Warning}>Changing your speed limit might cause you to drive faster than the laws allow you to in your country</Callout>
+                Login using your VanMoof account
+                <div className='formField'>
+                    <label htmlFor="email">Email</label>
+                    <input
+                        disabled={loading}
+                        id="email"
+                        value={login.email}
+                        onChange={e => setLogin(v => ({ ...v, email: e.target.value }))}
+                        placeholder="example@example.com"
+                    />
+                </div>
+                <div className='formField'>
+                    <label htmlFor='password'>Password</label>
+                    <input
+                        disabled={loading}
+                        id="password"
+                        value={login.password}
+                        onChange={e => setLogin(v => ({ ...v, password: e.target.value }))}
+                        type="password"
+                    />
+                </div>
+                <div className='loginBtn'>
+                    <P top={20}>
+                        <Button
+                            disabled={loading}
+                            type='submit'
+                        >Login</Button>
+                    </P>
+                </div>
+                <FormError error={error} />
+            </form>
+            <form className='loginForm' onSubmit={onSubmit} style={{ display: loginMethod === 'file' ? 'block' : 'none' }}>
+                <Callout kind={CalloutKind.Warning}>This website is <b>NOT</b> an offical VanMoof service/product</Callout>
+                <Callout kind={CalloutKind.Warning}>Changing your speed limit might cause you to drive faster than the laws allow you to in your country</Callout>
+                Login using your local JSON file
+                <div className='formField'>
+                    <label htmlFor='file'>Select JSON file</label>
+                    <input
+                        disabled={loading}
+                        id='file'
+                        type='file'
+                        accept='.json'
+                        onChange={onFileChange}
+                    />
+                </div>
+                <div className='loginBtn'>
+                    <P top={20}>
+                        <Button
+                            disabled={loading}
+                            type='submit'
+                        >Login</Button>
+                    </P>
+                </div>
+                <FormError error={error} />
+            </form>
+            <div className='loginMethod'>
```

Comment:
Wat dacht je van zoiets?
Zodra je dan op _"Login using JSON file"_ klikt opened die dan direct het select file modal en na het selecteren van een file gebruikt deze direct om in te loggen (gebruiker hoeft dan ook niet meer te klikken op login)

<img width="513" alt="image" src="https://github.com/mjarkk/vanmoof-web-controller/assets/15320763/cf5a0492-824c-4ab6-820a-386f85e10477">

Reply:
Ja, dat is een betere oplossing 👍🏻